src/main/java/core/basesyntax/WorkWithFile.java

```diff
@@ -1,8 +1,54 @@
 package core.basesyntax;
 
+import java.io.BufferedReader;
+import java.io.BufferedWriter;
+import java.io.File;
+import java.io.FileReader;
+import java.io.FileWriter;
+import java.io.IOException;
+
 public class WorkWithFile {
 
     public void getStatistic(String fromFileName, String toFileName) {
-
+        int supply = 0;
+        int buy = 0;
+        try {
+            BufferedReader bufferedReader = new BufferedReader(new FileReader(fromFileName));
```

Comment:
https://mate-academy.github.io/jv-program-common-mistakes/java-core/builder-file/work-with-file#try-with-resources

Reply:
please see the common mistakes file more attentive